### DISCUSSION TITLE
Question about Quartz with Azure Kubernetes Service

### DISCUSSION THREAD

**Discussion by Ghyath-Serhal** [quartznet/quartznet#1885] [Q&A]
It is not a feature and not an issue, I have a question regarding Quartz when deployed with Azure Kubernetes Service.
I have read the below, in this article
[https://learn.microsoft.com/en-us/azure/developer/java/migration/migrate-wildfly-to-wildfly-on-azure-kubernetes-service#determine-whether-your-application-relies-on-scheduled-jobs](url)

![aks](https://user-images.githubusercontent.com/6513173/209820661-3c1f59ce-2841-4852-aee3-b3c74d96a840.png)

I do not know why the problem can occur as they are mentioning since the management of job firing is handled by Quartz and not by AKS.
I did some testing, everything is working fine so far.

Please advise.

**Comment by xi23da** [comment]
您好，您发的邮件已经收到，请放心

**Comment by Ghyath-Serhal** [comment]
I have implemented and deployed on AKS. It has been running smoothly for over a year without any issues
